samples/oauth_rails_vethealth/app/models/health_api_response.rb

```diff
@@ -0,0 +1,46 @@
+class HealthApiResponse
+  attr_reader :api_name, :target, :page, :count
+  def initialize(api_name:, id:, access_token:, action: :read, page: 1, count: 10)
+    @api_name, @id, @action = api_name, id, action
+    if action == :search
+      @page, @count = page, count
+      search_param = (api_name == 'Patient' ? '_id' : 'patient')
+      @target = "https://dev-api.va.gov/services/argonaut/v0/#{api_name}?#{search_param}=#{id}&page=#{page}&_count=#{count}"
+    elsif action == :read
+      @page, @count = nil, nil
+      @target = "https://dev-api.va.gov/services/argonaut/v0/#{api_name}/#{id}"
+    else
+      raise "action must be one of the symbols :search or :read"
+    end
+    @api_response = HTTParty.get(@target, headers: { Authorization: "Bearer #{access_token}" })
```

Comment:
Please add line breaks before and after your if blocks (makes it a little easier to read)